src/Symfony/PasswordAuthenticatedUserStubFilesExtension.php

```diff
@@ -0,0 +1,23 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Symfony;
+
+use PHPStan\PhpDoc\StubFilesExtension;
+use function interface_exists;
+
+class PasswordAuthenticatedUserStubFilesExtension implements StubFilesExtension
+{
+
+	public function getFiles(): array
+	{
+		if (!interface_exists('Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface')) {
```

Comment:
Runtime concerns are irrelevant for PHPStan, you need to ask ReflectionProvider. If ReflectionProvider cannot be injected, then ask Reflector, as phpstan-doctrine does: https://github.com/phpstan/phpstan-doctrine/blob/1.3.x/src/Stubs/Doctrine/StubFilesExtensionLoader.php

Reply:
Not that I understand anything about PHPStan internals, but isn't that the "same" as in : https://github.com/phpstan/phpstan-symfony/blob/1.2.x/src/Symfony/InputBagStubFilesExtension.php#L13 ?

Reply:
That is reaaaaaally old and should be fixed too.

Reply:
Done for both StubFilesExtension.